SEP-0012.md

```diff
@@ -0,0 +1,495 @@
+This SEP should therefore also be considered as a discussion of future changes to the `Map` API, as the API proposed here would eventually (with long deprecation) be adopted by map for the functionality provided here.
+That being said, details of the migration of Map to this API are out of scope for this SEP, just a discussion of the eventual API to be adopted.
+
+## Background
+
+The creation of ndcube was originally motivated by the desire to represent and interrogate multi-dimensional solar datasets, such as those collected with the IRIS satellite.
+The fundamental objective of the package's primary class, `NDCube`, is to provide a user interface to this data which would **unite the data with the coordinate information describing the data**.
+The benefit of `NDCube` is its ability to do array-like operations which also update the coordinate system.
+In addition, `NDCube` works hard to provide information to the user that describes their data based on the coordinate system information.
+`NDCube` also provides simple summary visualisation support included in the object.
+This uses matplotlib to produce quick, coordinate-aware, and interactive visualisations of the data.
+The final major component added in `NDCube` is the concept of "extra coords" which allows the specification and integration with coordinates describing the data not specified by the primary WCS for the cube.
+
+## ndcube 2
+
+ndcube 1.0 was built on the functionality which existed in astropy at the time and provides a very useful foundation for tools such as the sunraster package as well as other efforts to handle multi-dimensional data in Python.
+During the process of writing NDCube 1.0 a lot of potential functionality which could be added to astropy to improve the WCS support was identified.
+In addition to this in-between the release of ndcube 1.0 and the development of 2.0, various long running projects such as the acceptance and implementation of [APE 14](https://github.com/astropy/astropy-APEs/blob/master/APE14.rst) were completed.
+This means that once many of these brand new features had been implemented in Astropy and gWCS it provided an excellent opportunity to reassess the state of NDCube and to rebase it onto this new WCS functionality.
+
+The main motivation for updating ndcube to use the APE 14 interface was to provide support for gWCS which is being used by the DKIST user tools.
+A lot of the other changes presented in this SEP are a consequence of this decision.
+
+
+## Summary of Changes from ndcube 1
+
+The fundamental change since the original version of ndcube is that the WCS type has been changed to be the High Level API as described by astropy's APE 14.
+This opens a lot of possibilities to the future uses of `NDCube` as the APE 14 API allows the coordinate system to be described by almost any functional transform.
+This could be the most common FITS WCS standard, or gWCS or custom implementations for specific facilites with different requirements.
```

Comment:
```suggestion
This could be the commonly used FITS WCS standard, gWCS, or custom implementations for specific facilities with different requirements.
```